packages/client/src/helpers/fetchApi.js

```diff
@@ -7,7 +7,7 @@ function getDefaultHeaders() {
 }
 
 export function addOrganizationId(url) {
-  return `${process.env.VUE_APP_GRANTS_API_URL}${url.replace(':organizationId', store.getters['users/selectedAgencyId'])}`;
+  return url.replace(':organizationId', store.getters['users/selectedAgencyId']);
```

Comment:
Is 'store' a way to get cookies? Or some other form of storage in the browser?

Reply:
We use the [VueX](https://v3.vuex.vuejs.org/) state management library. `store` refers to the VueX store, which manages clientside application state. Most of it is not persisted and gets refetched from the server on each page load, but there are a few parts that get stored in `localStorage` as well. The getter referenced here is defined [here](https://github.com/usdigitalresponse/usdr-gost/blob/_staging/packages/client/src/store/modules/users.js#L21).

This `organizationId` param in API URLs is used to support functionality in GOST where admins can impersonate any of their subagencies. This helper makes it so that every place that does XHR doesn't have to worry about which agency is being impersonated.